presto-server-rpm/src/main/rpm/preinstall

```diff
@@ -64,10 +64,9 @@ if [ "$java_found" = false ]; then
 +======================================================================+
 |      Error: Required Java version could not be found                 |
 +----------------------------------------------------------------------+
-| Please download the latest Oracle JDK/JRE from the Java web site     |
-|       > http://www.oracle.com/technetwork/java/javase/downloads <    |
+| Please download the latest OpenJDK or Oracle JDK/JRE.                |
 |                                                                      |
-| Presto requires Java 1.8 update 161 (8u161)                          |
+| Presto requires Java 1.8 update 161 (8u161) or Java 11.              |
```

Comment:
is `Java 11 >= Java 1.8`? If so then we could Presto requires at least Java 1.8 update 161. Notice that running Presto with Java 1.8 is not recommended. We could also mention that.

Reply:
Rephrased message.